elk/plotting/visualize.py

```diff
@@ -1,9 +1,6 @@
-from __future__ import annotations
```

Comment:
@norabelrose Could you explain why you removed this? I added this import following the advice of https://stackoverflow.com/questions/33533148/how-do-i-type-hint-a-method-with-the-type-of-the-enclosing-class